### PR TITLE
Update vercel-edge-config.md

### PR DESCRIPTION
**Pull request by cdruxerman**
Addressing comments B1, B2, B3, B5

### CODE REVIEW

File: docs/integrations/vercel-edge-config.md [hunk 1]
```diff
@@ -32,14 +32,16 @@ If your Edge Config cannot store your project's configuration after the integrat
...
+
 ### Configure Integration
 To get started, visit the [integration's page on Vercel](https://vercel.com/integrations/devcycle) and click "Add Integration".
 This will open a separate window which will load the setup form on the DevCycle dashboard. If you don't already have a DevCycle account,
 you will be taken through the signup process. Once you've created your account and Organization, you'll be brought back to the integration
 setup.
 
 Once logged in, you can select which projects in DevCycle you want to sync to Edge Config, and which Edge Config to sync them to.
-You must have created an Edge Config beforehand in order to select it here. To create an Edge Config, follow [Vercel's documentation](https://vercel.com/docs/storage/edge-config/get-started).
+If you don't have an Edge Config already set up, you'll have the opportunity to set one up in the configuration flow. You can also create an Edge Config ahead of time by following [Vercel's documentation](https://vercel.com/docs/storage/edge-config/get-started).
```
Comment (ajwootto) [inline docs/integrations/vercel-edge-config.md:44]: this isnt true, the edge config will be created automatically with the name "devcycle"

File: docs/integrations/vercel-edge-config.md [hunk 2]
```diff
@@ -14,8 +14,8 @@ The Edge Config integration is currently available for the following SDKs:
 - Node.js
 - Next.js
 
-Using the integration requires a Vercel account and an application running in that account which is using 
-one of the above SDKs.
+Using the integration requires a Vercel account, an application running in that account which is using 
+one of the above SDKs, and an Edge Config connected to a project in Vercel..
```
Comment (ajwootto) [inline docs/integrations/vercel-edge-config.md:18]:
```suggestion
one of the above SDKs, and an Edge Config connected to a project in Vercel.
```

File: docs/integrations/vercel-edge-config.md [hunk 3]
```diff
@@ -14,8 +14,8 @@ The Edge Config integration is currently available for the following SDKs:
 - Node.js
 - Next.js
 
-Using the integration requires a Vercel account and an application running in that account which is using 
-one of the above SDKs.
+Using the integration requires a Vercel account, an application running in that account which is using 
+one of the above SDKs, and an Edge Config connected to a project in Vercel.
```
Comment (ajwootto) [inline docs/integrations/vercel-edge-config.md:18]:
```suggestion
one of the above SDKs, and an Edge Config [connected to a project in Vercel](https://vercel.com/docs/storage/edge-config/get-started#connect-your-vercel-project).
```

File: docs/integrations/vercel-edge-config.md (hunk 1 above)
Comment (cdruxerman) [reply docs/integrations/vercel-edge-config.md]: edited

File: docs/integrations/vercel-edge-config.md [hunk 4]
```diff
@@ -32,14 +32,20 @@ If your Edge Config cannot store your project's configuration after the integrat
 
 ## Setup
 
+### DevCycle
+If you haven't already installed the DevCycle [Node.js](https://docs.devcycle.com/sdk/server-side-sdks/node/) or [Next.js](https://docs.devcycle.com/sdk/server-side-sdks/nestjs/) SDK you can follow the installation and usage guides for those SDKs in our [documentation here](https://docs.devcycle.com/sdk/). You can also find helpful setup information like where to find SDK keys in our [Quickstart Tutorial](https://docs.devcycle.com/introduction/quickstart).
+
+###Vercel
```
Comment (ajwootto) [inline docs/integrations/vercel-edge-config.md:38]: move this to inside the "Setup SDK" section cuz by that point they'll have the integration set up and are about to start the code part, plus we'll have created an edge config for them that they can connect

Comment (cdruxerman) [reply docs/integrations/vercel-edge-config.md]: i think i moved the part you wanted moved. take another look

Comment (ajwootto) [reply docs/integrations/vercel-edge-config.md]: it appears to be in the same place as before

Comment (ajwootto) [reply docs/integrations/vercel-edge-config.md]: i was talking about moving the Vercel project connection part

Comment (cdruxerman) [reply docs/integrations/vercel-edge-config.md]: 🤦

**Review by ajwootto — APPROVED** [review]

**Pull request merged**